dql/cli.py

```diff
@@ -396,6 +397,16 @@ def complete_opt_allow_select_scan(self, text, *_):
         """ Autocomplete for allow_select_scan option """
         return [t for t in ("true", "false", "yes", "no") if t.startswith(text.lower())]
 
+    def opt_output_json_as_list(self, output):
+        """ Set option output_json_as_list """
+        output = output.lower() in ("true", "t", "yes", "y")
+        self.conf["output_json_as_list"] = output
+        self.engine.output_json_as_list = output
```

Comment:
You may want to set the value on the engine during startup as well, with the value loaded from the config. Probably around here: https://github.com/stevearc/dql/blob/6b0ab8df07afd226816937ee98b4ef73187206fc/dql/cli.py#L202